RHEL/7/input/xccdf/system/network/kernel.xml

```diff
@@ -70,7 +76,7 @@ only appropriate for systems acting as routers.</rationale>
 acting as either hosts or routers to improve the system's ability defend
 against certain types of IPv4 protocol attacks.</description>
 
-<Value id="sysctl_net_ipv4_conf_all_accept_source_route_value" type="number"
+<Value id="sysctl_net_ipv4_conf_all_accept_source_route_value" type="string"
```

Comment:
@shawndwells 
This is a result of previous XCCDF to OVAL data export constraint, introduced due to ScapVal NIST sanity checks.
Besides other things the ```relabelids.py``` transformation / helper script now performs if the constraint isn't violated. In case it is (XCCDF ```@type``` doesn't match expected OVAL ```@datatype```) it will issue a warning listing particular ```<xccdf:Value>``` which broke the constraint:
  https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/transforms/relabelids.py#L144

And also (to ensure build content pass the NIST ScapVal requirements) it will correct the ```@type```` attribute of ```<xccdf:Value>``` to the corresponding type as required by ```@datatype``` attribute of bound OVAL variable (this approach is used, because OVAL variable can have further variables derived from the external one, and changing external OVAL variable datatype might break ```make validate``` target for that product).

Just explanation of the above behaviour.